### ISSUE TITLE
Basic features implemented

### ISSUE THREAD

**Issue by chinma-yyy** [chinma-yyy/botpress-demo#1]
## These are basic features i have tried to implement

[Screencast from 03-04-23 10:10:49 PM IST.webm](https://user-images.githubusercontent.com/112387862/229493634-60609edb-7c18-45fd-93b9-3de4159adef2.webm)

**Comment by syedfawzulazim** [comment]
Hi, Chinma is there any way I can communicate with you?

**Comment by chinma-yyy** [comment]
Yes you can view any platform for as per your convenience from https://www.chinmayyy.tech

**Comment by syedfawzulazim** [comment]
I tried your LinekdIn to connect,  basically I think you are doing a great job contributing to rocket chat. I am new to open source contribution,  I liked this project but  do not know how to start. I would be very much helpfully if you consider helping me on this. Thanks a lot.
